recipes/assimp/5.x/conanfile.py

```diff
@@ -107,6 +107,10 @@ def _source_subfolder(self):
     def _build_subfolder(self):
         return "build_subfolder"
 
+    @property
+    def _is_msvc(self):
+        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
+
```

Comment:
It can be replaced by [conan.tools.microsoft.is_msvc(self)](https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc)

It's already available on CCI, as it is running 1.45 now: https://github.com/conan-io/conan-center-index/blob/master/docs/changelog.md#31-march-2022---1155-cest